crates/holochain_state/src/integrate.rs

```diff
@@ -98,8 +98,20 @@ fn insert_locally_validated_op(
 
     let op_type = op.get_type();
 
+    let serialized_size = std::cmp::min(
+        op.as_content()
+            .as_chain_op()
+            .and_then(|op| {
+                holochain_serialized_bytes::encode(&op)
+                    .map(|e| e.len())
+                    .ok()
+            })
+            .unwrap_or_default(),
+        u32::MAX as usize,
+    );
```

Comment:
Not following the std::cmp::min here... the unwrap_or_default will default to zero, right? so this will never choose u32::MAX?

Reply:
Oh, I see it's cast to a usize, so we're just keeping it below u32::MAX if usize is 64 bits.

Reply:
I'm trying to be careful about the size being a u64 and casting down to u32 failing. Maybe this isn't the right way to deal with that. Or maybe I shouldn't be worrying about it because that would be a silly size

Reply:
More importantly you're filtering the ops above, so this case can never happen anyway, right?

Reply:
Good point. This is for the local insert but that does mean it's been through sys validation and the entry size is limited. So in the incoming case, we check the size and in the authored case we have checked too. No need to check again here.

I've left a comment about why this cast is safe and also updated the docs in the sys validation workflow because there was something missing there for StoreRecord